.travis.yml

```diff
@@ -17,11 +38,11 @@ matrix:
   - os: linux
     compiler: clang
     env: Q_OR_C_MAKE=cmake
-
```

Comment:
As the above combination now also works (because we are no longer using the bugged Qt 5.0.2) please remove the lines beginning `matrix:`down to here to remove allowing the one test build (CMake/Clang/Linux tuple) to fail and still count the overall C.I. as a :white_check_mark: "pass".